feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/metadata.textproto

```diff
@@ -5,3 +5,16 @@ uuid:  "cf3f42af-1463-4a3b-b691-bf06edc3db14"
 plan_id:  "DP-1.19"
 description:  "Egress traffic DSCP rewrite"
 testbed:  TESTBED_DUT_ATE_2LINKS
+
+platform_exceptions: {
+  platform: {
+    vendor: ARISTA
+  }
+  deviations: {
+    interface_enabled: true
+    default_network_instance: "default"
+    qos_remark_oc_unsupported: true
+    static_mpls_lsp_oc_unsupported: true
+    gue_gre_decap_unsupported: true
+  }
+}
```

Comment:
Do we have a bug for each deviation? Is there a plan to resolve the unsupported OC and features?

Reply:
Yes we have bug #409229653 #415889077 #409183165.. Since these are already comminted with metadata.proto file, not commiting it again..